test/scripts/jenkins_build_kibana.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+source src/dev/ci_setup/setup_env.sh
+yarn run grunt functionalTests:ensureAllTestsInCiGroup;
+node scripts/build --debug --oss
+
+node scripts/es snapshot --license=oss --download-only;
```

Comment:
Why is this necessary? We should only need this in the packer cache, though maybe it makes sense to download the current version before the tests start anyway, so we fail faster when snapshots aren't available for some reason.

Reply:
When the version needed doesn't match what's in the packer cache, all of the workers try to download the snapshot at the same time, and there are some concurrency issues with how it's currently set up. So, I'm just downloading it once before all of the tests start